test/SolverInterface.cpp

```diff
@@ -153,6 +153,13 @@ bool SolverInterface::hasToEvaluateFineModel() const
   return 0;
 }
 
+bool SolverInterface:: isMeshConnectivityRequired
+(
+  int           meshID )
```

Comment:
```suggestion
  int           meshID ) const
```
I added a `const` here and now it works.